python/ray/util/annotations.py

```diff
@@ -107,12 +112,19 @@ def Deprecated(*args, **kwargs):
     if len(args) == 1 and len(kwargs) == 0 and callable(args[0]):
         return Deprecated()(args[0])
 
-    message = (
+    doc_message = (
         "\n    DEPRECATED: This API is deprecated and may be removed "
         "in future Ray releases."
     )
+    warning_message = (
+        "This API is deprecated and may be removed in future Ray releases."
+    )
+
+    warning = kwargs.pop("warning", False)
```

Comment:
Default to false to reduce the risk of cherry pick and only enable it for newly deprecated apis in 2.0. Some of the existing deprecated apis have their custom way of printing the warning. We should unify them post 2.0.

Reply:
Since we decide not to cherry pick this, should we fix it now?